conftest.py

```diff
@@ -79,6 +79,7 @@ def setup_host(hostname_or_ip):
 def hosts(request):
     # a list of master hosts, each from a different pool
     hostname_list = request.param.split(',')
+    print(hostname_list)
```

Comment:
Could you remove this line? :) It's not directly related to the PR.

Reply:
Absolutely yes :)